### DISCUSSION TITLE
AI科技執法

### DISCUSSION THREAD

**Discussion by number32984** [1101-GS3073-C/Python-Community-GS3073-C#612] [General]
我們先創造垃圾，在創造產品來限制我們創造垃圾，這個IDEA真先進
https://www.gvm.com.tw/article/97515

**Comment by larry40305** [comment]
以後就會有偵測是否有人亂丟垃圾的AI了。

**Comment by magicmicheal** [comment]
黑煙、白煙都逃不過AI法眼，之後應該可以推出偵測空氣品質的AI來抓二手菸

**Comment by awkward-willy** [comment]
隨著科技的發展以及不斷都市化的結果，越來越多的汽機車出現在街道上，這樣的科技能快速且精準的抓出應做檢測的車輛，或許未來若能像測速照相等儀器普及，就能大幅改善都市空汙的問題，當然也可以結合更多功能，如上面有人提到的抓二手菸，或者抓惱人的改管機車騎士。

**Comment by manipulateed** [comment]
終於，每次看到別人的汽車或是機車起步時冒出濃濃的煙，一整天的好心情都沒有了，這個科技我覺得很有必要強制執行，還給我們一個更乾淨的空氣與都市品質。

**Comment by 0212Jerry** [comment]
之前有看過類似的新聞，一群機車因為禮讓救護車而被科技執法全部開罰單，可見科技執法還是要再多加訓練，面對不同的情境需要有所校正

**Comment by Vincent20030314** [comment]
科技執法還要發展一段時間再使用，否則可能會有很多爭議。
